core/src/main/resources/com/graphhopper/util/zh_TW.txt

```diff
@@ -1,82 +1,82 @@
 # do not edit manually, instead use spreadsheet https://t.co/f086oJXAEI and script ./core/files/update-translations.sh
 
-continue=繼續
-continue_onto=繼續行駛到 %1$s
-turn_left=左轉
-turn_right=右轉
-turn_slight_left=微靠左轉
-turn_slight_right=微靠右轉
-turn_sharp_left=左急轉
-turn_sharp_right=右急轉
-keep_left=保持左側
-keep_right=保持右側
-turn_onto=%1$s 進入%2$s
-u_turn=迴轉
-unknown=未知指示標誌 '%1$s'
-web.search_button=搜尋
-web.from_hint=起點
-web.via_hint=途經
-web.to_hint=迄點
-web.more_button=更多
-web.gpx_export_button=匯出GPS
-web.route_info=%1$s 需時 %2$s
-web.pt_route_info=於 %1$s 抵達，%2$s 次轉乘 (%3$s)
-web.pt_route_info_walking=於 %1$s 抵達，僅步行 (%2$s)
-web.locations_not_found=無法進行規劃。無法在此區域內找到指定的地點
-web.bike=自行車
-web.racingbike=競技自行車
-web.mtb=登山車
-web.car=汽車
-web.foot=步行
-web.hike=健行
-web.small_truck=小貨車
-web.bus=公車
-web.truck=貨車
-web.staticlink=永久鏈結
-web.motorcycle=摩托車
-via=途經
-finish=抵達目的地
-hour_abbr=小時
-day_abbr=天
-min_abbr=分鐘
-km_abbr=公里
-m_abbr=公尺
-mi_abbr=英里
-ft_abbr=英尺
-road=道路
-off_bike=下自行車
-cycleway=自行車道
-way=路
-small_way=小路
-paved=路面有鋪設
-unpaved=路面無鋪設
-stopover=中途點 %1$s
-roundabout_enter=進入圓環
-roundabout_exit=於 %1$s 個出口離開圓環
-roundabout_exit_onto=於 %1$s 個出口離開圓環，進入 %2$s
-total_ascend=總共上昇 %1$s
-total_descend=總共下降 %1$s
-way_contains_ford=路徑中含有淺灘
-way_contains_ferry=搭乘渡輪
-way_contains_private=私人道路
-way_contains_toll=收費道路
-pt_start_trip=進入 %1$s
-pt_end_trip=離開 %1$s
-pt_transfer_to=變換至 %1$s
-web.start_label=出發點
-web.intermediate_label=途經點
-web.end_label=抵達點
-web.set_start=設為出發點
-web.set_intermediate=設為途經點
-web.set_end=設為抵達點
-web.center_map=設為地圖中心
-web.show_coords=顯示坐標
-web.route=路線
-web.delete_from_route=從路線中移除
-web.marker=標記
-web.gh_offline_info=GraphHopper API 離線狀態？
-web.refresh_button=刷新頁面
-web.server_status=狀態
-web.zoom_in=放大
-web.zoom_out=縮小
+continue=繼續
+continue_onto=繼續行駛到 %1$s
+turn_left=左轉
+turn_right=右轉
+turn_slight_left=微靠左轉
+turn_slight_right=微靠右轉
+turn_sharp_left=左急轉
+turn_sharp_right=右急轉
+keep_left=保持左側
+keep_right=保持右側
+turn_onto=%1$s 進入%2$s
+u_turn=迴轉
+unknown=未知指示標誌 '%1$s'
+web.search_button=搜尋
+web.from_hint=起點
+web.via_hint=途經
+web.to_hint=迄點
+web.more_button=更多
+web.gpx_export_button=匯出GPS
+web.route_info=%1$s 需時 %2$s
+web.pt_route_info=於 %1$s 抵達，%2$s 次轉乘 (%3$s)
+web.pt_route_info_walking=於 %1$s 抵達，僅步行 (%2$s)
+web.locations_not_found=無法進行規劃。無法在此區域內找到指定的地點
+web.bike=自行車
+web.racingbike=競技自行車
+web.mtb=登山車
+web.car=汽車
+web.foot=步行
+web.hike=健行
+web.small_truck=小貨車
+web.bus=公車
+web.truck=貨車
+web.staticlink=永久鏈結
+web.motorcycle=摩托車
+via=途經
+finish=抵達目的地
+hour_abbr=小時
+day_abbr=天
+min_abbr=分鐘
+km_abbr=公里
+m_abbr=公尺
+mi_abbr=英里
+ft_abbr=英尺
+road=道路
+off_bike=下自行車
+cycleway=自行車道
+way=路
+small_way=小路
+paved=路面有鋪設
+unpaved=路面無鋪設
+stopover=中途點 %1$s
+roundabout_enter=進入圓環
+roundabout_exit=於 %1$s 個出口離開圓環
+roundabout_exit_onto=於 %1$s 個出口離開圓環，進入 %2$s
+total_ascend=總共上昇 %1$s
+total_descend=總共下降 %1$s
+way_contains_ford=路徑中含有淺灘
+way_contains_ferry=搭乘渡輪
```

Comment:
Ugh looks like this file had different line endings...

Reply:
I think the problem is in our core/files/update-translations.sh file that grabs the file from google. And likely because zh_TW.txt is the last language the line endings of the original file are included.

Reply:
Ok I reverted the file its not related here